challenges/objects-arrays.js

```diff
@@ -6,6 +6,51 @@
   object name, diet, weight, length, period
 */
 
+class Dinosaurs {
```

Comment:
This assignment did not ask to make use of classes, but great job doing so! Be sure to always read the instructions and don't over-complicate things for yourself